core-plugins/docs/HDFSAction-action.md

```diff
@@ -0,0 +1,53 @@
+# HDFS Action
+
+
+Description
+-----------
+Moves a file or files within an HDFS cluster.
+
+
+Use Case
+--------
+This action can be used when a file or files need to be moved to a new location in an HDFS cluster, 
+often required when archiving files.
+
+
+Properties
+----------
+**sourcePath:** The full HDFS path of the file or directory that is to be moved. In the case of a directory, if
+fileRegex is set, then only files in the source directory matching the wildcard regex will be moved.
+Otherwise, all files in the directory will be moved. For example: `hdfs://hostname/tmp`.
+
+**destPath:** The valid, full HDFS destination path in the same cluster where the file or files are to be moved.
+For moving a single file, this means that all parent directories must already exist. For moving multiple files in a
+directory or an entire directory, the path to the desired directory is passed and that directory and all parent
+directories must already exist.
+
+**fileRegex:** Wildcard regular expression to filter the files in the source directory that will be moved.
+
+**continueOnError:** Indicates if the pipeline should continue if the move process fails.
+
+
+Example
+-------
+This example moves a file from `/source/path` to `/dest/path`:
+
+{
+    "name": "HDFSAction",
+    "plugin": {
+        "name": "HDFSAction",
+        "type": "action",
+        "label": "HDFSAction",
```

Comment:
should remove label, as it is something the UI injects, it is not used by the backend anywhere (UI shouldn't be putting it there).